addons/sale_timesheet/models/sale_order.py

```diff
@@ -236,6 +236,10 @@ def _timesheet_create_project(self):
                 'partner_id': self.order_id.partner_id.id,
                 'email_from': self.order_id.partner_id.email,
             })
+            # duplicating a project doesn't set the SO on sub-tasks
+            project.tasks.filtered(lambda task: task.parent_id != False).write({
+                'sale_line_id': self.id,
```

Comment:
is the partner and partner email already written on (sub)tasks ?
We need the partner to be set, otherwise sale_line field will not be displayed on form view ;)

Reply:
They are 
![19-05-16_11-44-44](https://user-images.githubusercontent.com/8208953/57844283-0c9c8d00-77d0-11e9-8fa1-ea084d65c5d9.png)